src/api/firebase.js

```diff
@@ -197,8 +200,41 @@ export async function updateItem(listPath, { itemId, totalPurchases }) {
 
 	const itemDoc = doc(db, listPath, 'items', itemId);
 
+	const today = new Date();
```

Comment:
I created a PR to showcase this but I would try to separate logic into separate functions. It's a bit easier to show in a separate PR: 

https://github.com/the-collab-lab/tcl-67-smart-shopping-list/pull/47